namespaces/cloud-platform-live-0.k8s.integration.dsd.io/disclosure-checker-staging/resources/ecr.tf

```diff
@@ -0,0 +1,23 @@
+#####################################
+# Disclosure Checker ECR repository
+#####################################
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
```

Comment:
Please use the latest version of this module: 2.1

Reply:
updated to version 2.1